doc/manual/methods.rst

```diff
@@ -226,148 +235,29 @@ values is perhaps the single most powerful and central feature of the
 Julia language. Core operations typically have dozens of methods:
 
 .. doctest::
+   :options: +SKIP
```

Comment:
Fair enough. If we do want to require doctests to pass for PRs I think this particular doctest is going to be the main issue.